src/lib/draw-and-pick.js

```diff
@@ -265,6 +265,17 @@ function getClosestFromPickingBuffer(gl, {
   deviceY,
   deviceRadius
 }) {
+  let closestResultToCenter = {
+    pickedColor: EMPTY_PIXEL,
+    pickedLayer: null,
+    pickedObjectIndex: -1
+  };
```

Comment:
While I realize you just moved this around, shouldn't this be an `EMPTY_PICK_INFO` constant at the top of the file?

And aren't there more fields in a pickInfo object [(doc link)](https://github.com/uber/deck.gl/blob/master/docs/get-started/interactivity.md#the-picking-info-object) that should be initialized to default values? (if there are no reasonable defaults, setting to `null` is better than leaving as `undefined` in my view as it shows that this was an explicit, intentional choice).

Reply:
This object is not the `pickInfo` object. It is only used by the other functions in draw-and-pick to create `pickInfo`.